proposals/lip-0047.md

```diff
@@ -747,7 +745,7 @@ setGeneratorList(addresses)
 where:
 
 * `registerValidatorKeys` is defined in the [Validators module][lip-0044#register-validator-keys].
-* `setBFTParameters` is a function exposed by the [BFT module][lip-0058].
+* `setValidatorParams` is a function exposed by the [Validators module][lip-0044#setvalidatorparams].
 * `setGeneratorList` is a function exposed by the [Validators module][lip-0044].
```

Comment:
Ah, only after your comment I have noticed that this function is no longer defined in the Validators module and has to be removed.

Reply:
Ah, I see. I was at first confused because I thaught I see a version of the Validators module that had this function defined, but I haven't realized that it wasn't the final one.